src/components/CommentContainer/CommentContainer.tsx

```diff
@@ -130,7 +130,7 @@ export const CommentContainer = ({
         getMoreResponses(commentId)
             .then(json => {
                 setExpanded(true);
-                setResponses(json.comment.responses || []);
+                setResponses([...responses, ...(json.comment.responses || [])]);
```

Comment:
I think the problem here is that `json.comment.responses` is not (no longer?) being returned. I'm not sure that adding `...responses` will provide the replies.